src/lib/ai/tools/get-weather.ts

```
@@ -1,18 +1,18 @@
-  }),
-  execute: async ({ latitude, longitude }) => {
-    const response = await fetch(
-      `https://api.open-meteo.com/v1/forecast?latitude=${latitude}&longitude=${longitude}&current=temperature_2m&hourly=temperature_2m&daily=sunrise,sunset&timezone=auto`,
-    );
-
-    const weatherData = await response.json();
-    return weatherData;
-  },
-});
+import { tool } from 'ai';
+import { z } from 'zod/v4';
```

ESLint

[error] Unable to resolve path to module 'zod/v4'.

(import-x/no-unresolved)